WalletWasabi.Fluent/Models/Wallets/CoinModel.cs

```diff
@@ -10,17 +11,36 @@ namespace WalletWasabi.Fluent.Models.Wallets;
 [AutoInterface]
 public partial class CoinModel : ReactiveObject
 {
+	[AutoNotify] private bool _isExcludedFromCoinJoin;
+	[AutoNotify] private int _anonymitySet;
+	[AutoNotify] private bool _confirmed;
+	[AutoNotify] private bool _isCoinJoinInProgress;
+	[AutoNotify] private bool _isBanned;
+	[AutoNotify] private string? _bannedUntilUtcToolTip;
+	[AutoNotify] private string? _confirmedToolTip;
```

Comment:
I am not sure about this btw. ViewModel should be the bridge between the view and the model, but now we directly bind to the model from the view.

Reply:
@soosr this is an interesting discussion to have.

I don't think we should attach ourselves to any sort of dogma, and instead do what's best for us.

In this particular sense, we have two options:

1 - Avoid binding directly to Model and force every ViewModel to intermediate between the Model and the View, therefore "copying" all properties from the Model into the ViewModel.

This approach might look "purer" from a dogmatic perspective, but in practice it will lead to lots of duplicated code, lots of `model.WhenAnyValue(x => x.Prop).BindTo(this, x => x.Prop);` boilerplate, increased memory usage, and the need to go through many "layers" of code whenever one wants to add an additional piece of data in the UI (having to add the same property both in the Model and the ViewModel), effectively reducing maintainability.

2 - Allow binding directly to Model whenever it makes sense (such as for read-only data, TreeDataGrid, etc)

This approach might look "dirtier" but will remove all the downsides mentioned above.

I don't want to impose either option, so everyone in @zkSNACKs/ui-team is invited to participate in this discussion

There might be other upsides and downsides to either approach I haven't considered here.

Reply:
@wieslawsoltes Can you elaborate a bit? I'm not sure I follow. Both ViewModel and Model are INPC (ReactiveObject really). Does the issue you mention still occur in that case?